thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2799,6 +2805,7 @@ void CMasterGraph::getFinalProgress()
             }
         }
     }
+    jobM->updateActiveSpillSize(graphSpillSize, totalActiveSpillSize);
```

Comment:
@jakesmith Will this be called more than once for a loop subgraph?  If so, will that cause potential problems.

Reply:
Yes, and related to https://github.com/hpcc-systems/HPCC-Platform/pull/16845#discussion_r1108208706,

Both 'graphSpillSize' and 'totalActiveSpillSize' should be reset at start of graph, to cater for the case where the graph is re-executed in a LOOP/GRAPH pattern.